tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -923,8 +980,28 @@ def test_get(agent, filter, opt=:raw)
 def test_set(agent, cmd)
   return if cmd.empty?
   logger.info(cmd)
-  cmd_prefix = PUPPET_BINPATH + "resource cisco_command_config 'cc' "
-  on(agent, cmd_prefix + "test_set='#{cmd}'")
+  if !agent.nil?
+    cmd_prefix = PUPPET_BINPATH + "resource cisco_command_config 'cc' "
+    on(agent, cmd_prefix + "test_set='#{cmd}'")
+  else
+
+    conf_dir = `puppet config print | grep deviceconfig`
```

Comment:
`config print` allows fetching a specific value: https://puppet.com/docs/puppet/6.0/config_print.html#to-see-the-value-of-one-setting, no need to grep.

Reply:
config print deviceconfig saves a regex parse

Reply:
Now using proper .cfg file, no need to puppet config hackiness